src/views/Home.vue

```diff
@@ -1,18 +1,72 @@
 <template>
   <div class="home">
-    <img alt="Vue logo" src="../assets/logo.png">
-    <HelloWorld msg="Welcome to Your Vue.js App"/>
+    <div class="form">
+      <label class="form__label">Nickname</label>
+      <input
+        class="form__input"
+        placeholder="Write here..."
+        type="text"
+        v-model="user"
+        @keyup="debounceFunc" />
+    </div>
+    <users-list :users="users"></users-list>
   </div>
 </template>
 
 <script>
-// @ is an alias to /src
-import HelloWorld from '@/components/HelloWorld.vue';
+import axios from 'axios';
+import UsersList from '@/components/UsersList.vue';
 
 export default {
   name: 'home',
   components: {
-    HelloWorld,
+    UsersList,
+  },
+  data() {
+    return {
+      user: '',
+      users: [],
+    };
+  },
+  methods: {
+    debounce(func, wait) {
+      let timeout;
+      return () => {
+        clearTimeout(timeout);
+        timeout = setTimeout(() => func(), wait);
+      };
+    },
+    getUsers() {
+      axios
```

Comment:
Please validate if `this.user` is not empty, so that the request is not called with empty params